src/components/Table/GridTable.tsx

```diff
@@ -1208,6 +1221,8 @@ function GridRow<R extends Kinded, S>(props: GridRowProps<R, S>): ReactElement {
           ...(isGridCellContent(maybeContent) && maybeContent.typeScale ? Css[maybeContent.typeScale].$ : {}),
           // And any cell specific css
           ...(isGridCellContent(maybeContent) && maybeContent.css ? maybeContent.css : {}),
+          // Apply cell highlight styles to active cell and hover
+          ...Css.bw1.bsSolid.br4.if(applyCellHighlight && isCellActive).bLightBlue900.else.bTransparent.$,
```

Comment:
**nit:** You do not need the `else` block here. You could _always_ set `bTransparent` and only override it with the `if` statement.